### PR TITLE
change win & mac buttons to text buttons

### PR DESCRIPTION
**Pull request by Simon-Laux**

### CODE REVIEW

File: _includes/download-boxes.html [hunk 1]
```diff
@@ -62,9 +60,7 @@ <h1 id="noscript-headline">{%if tx.menu.download != ""%}{{ tx.menu.download }}{%
                 <a href="https://www.microsoft.com/en-us/p/deltachat/9pjtxx7hn3pk?activetab=pivot:overviewtab" class="img-badge">
                     <img src="../assets/badges/microsoft.svg" alt="{%if tx.download.dl_microsoft != ""%}{{ tx.download.dl_microsoft }}{%else%}{{ txEn.download.dl_microsoft }}{%endif%}" />
                 </a>
-                <a href="https://download.delta.chat/desktop/v{{VERSION_DESKTOP}}/DeltaChat%20Setup%20{{VERSION_DESKTOP}}.exe" class="img-badge">
-                    <img src="../assets/badges/get-it-on-windows.png" alt="Get it on Windows" />
-                </a>
+                <a href="https://download.delta.chat/desktop/v{{VERSION_DESKTOP}}/DeltaChat%20Setup%20{{VERSION_DESKTOP}}.exe"><small>Download</small> Installer.exe</a>
```
Comment (r10s) [inline _includes/download-boxes.html]: i might be confusing showing the filename Installer.exe on the button when the final file is called *Setup.exe - so maybe better say "Download Setup.exe"

File: _includes/download-boxes.html [hunk 2]
```diff
@@ -48,9 +48,7 @@ <h1 id="noscript-headline">{%if tx.menu.download != ""%}{{ tx.menu.download }}{%
                 <a href="https://apps.apple.com/us/app/delta-chat-desktop/id1462750497" class="img-badge">
                     <img src="../assets/badges/mac-appstore.svg" alt="{%if tx.download.dl_mac_appstore != ""%}{{ tx.download.dl_mac_appstore }}{%else%}{{ txEn.download.dl_mac_appstore }}{%endif%}" />
                 </a>
-                <a href="https://download.delta.chat/desktop/v{{VERSION_DESKTOP}}/DeltaChat-{{VERSION_DESKTOP}}.dmg" class="img-badge">
-                    <img src="../assets/badges/get-it-on-macos.png" alt="Get it on macOS" />
-                </a>
+                <a href="https://download.delta.chat/desktop/v{{VERSION_DESKTOP}}/DeltaChat-{{VERSION_DESKTOP}}.dmg"><small>Download</small> DeltaChat.DMG</a>
```
Comment (r10s) [inline _includes/download-boxes.html]: i would not use large letters for DMG - the downloaded file will be DeltaChat*.dmg with small letters, so maybe just use "Download DeltaChat.dmg"

**Review by r10s — APPROVED** [review]
some suggestions only :)

**Comment by Simon-Laux** [comment]
feel free to change it as you wish and merge it. :smile:

**Comment by r10s** [comment]
merging, before that gets too stale ... ;)

**Pull request merged**